_posts/2019-06-30-GraphQL-Announcement.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+title: GraphQL Support for Parse Server
+date: 2019-06-25 22:10 -0000
+comments: true
+author: tomwfox
+categories: [Announcements, Update]
+---
+
+We're glad to announce that Parse Server now has support for GraphQL...
+
+<!-- more -->
+
+It contains a minimum set of queries and mutations to have a GraphQL API running on top of Parse Server, including: Auto Schema, Users Authentication, Files, Class Objects supporting all data types and CRUD operations for MongoDB and Postgres.
```

Comment:
This statement seems negative. Like something that is still not complete. Maybe something like this:
The x.x.x release automatically generates the main queries and mutations to have a GraphQL API running on top of ParseServer, ...